lib/Service/CardService.php

```diff
@@ -201,6 +204,11 @@ public function assignUser($cardId, $userId) {
 				return false;
 			}
 		}
+
+		/* Notify user about the card assignment */
+		$card = $this->cardMapper->find($cardId);
+		$this->notificationHelper->sendCardAssigned($card, $userId);
```

Comment:
We should only send a notification if the user assigns somebody else. If you assign yourself, there is no need for a notification.

Reply:
Done